reg/drone/service/common/Arming.0.1.uavcan

```diff
@@ -0,0 +1,52 @@
+# The arming state is used to command or report the availability (readiness) status of a networked service (subsystem).
+#
+# A simple system would usually have one arming command subject published by some central controller
+# that other components subscribe to. Said subject acts as a global power switch. Every subsystem controlled in
+# such way would usually report its arming status back to account for the fact that the transition between the arming
+# status may not be instantaneous. The arming status reporting is done by means of the service heartbeat type
+# that is also defined in this namespace; the service heartbeat type is a structural subtype of this arming type.
+#
+#   +------------+
+#   | Controller |----------+----------------+----------------+---------...     arming command subject
+#   +------------+          |                |                |
+#     ^   ^   ^             v                v                v
+#     |   |   |        +---------+      +---------+      +---------+
+#     |   |   |        | Service |      | Service |      | Service |    ...
+#     |   |   |        +---------+      +---------+      +---------+
+#     |   |   |             |                |                |
+#     |   |   +-------------+                |                |
+#     |   +----------------------------------+                |                 service heartbeat subjects
+#     +-------------------------------------------------------+
+#
+# In a less trivial use case there may be an arbitrary number of such arming command subjects (local power switches)
+# controlling various systems within the vehicle (e.g., propulsion, perception sensors, communication, etc).
+#
+# The publication rate is defined on a per-service basis, but it should never be lower than 1 Hz,
+# excepting services that are in the SLEEP state, in which case it is permitted to cease all network activity.
+
+truncated uint2 value
+
+uint2 SLEEP = 0
```

Comment:
Could there be some more detailed power management modes later, or should that be vendor specific? I'm thinking things like GPS, LoraWAN where we have several low power modes to pick from depending on how fast the system needs to come back up. This might fit better in a vendor command or register.

Reply:
> several low power modes to pick from depending on how fast the system needs to come back up

For future-proofing, we could reassign DISARMED=2 and ARMED=3, leaving the value 1 unused for now. In the future we could add a shallow-sleep mode, should the need arise. Should we do that?

Reply:
Sounds good to me